engine-rest/engine-rest-openapi/src/main/templates/models/org/camunda/bpm/engine/rest/dto/runtime/user/CreateUserDto.ftl

```diff
@@ -0,0 +1,20 @@
+<@lib.dto>
+
+    <@lib.property
+        name = "profile"
+        type = "object"
+        additionalProperties = true
+        dto = "UserDto"
```

Comment:
The type here should be `ref`, we don't need `additionalProperties` and DTO can be renamed as we discussed:
```suggestion
        type = "ref"
        dto = "UserProfileDto"
```